internal/cmd/users.go

```diff
@@ -0,0 +1,35 @@
+package cmd
+
+import (
+	"strings"
+
+	"github.com/edulinq/autograder/internal/api/core"
+	"github.com/edulinq/autograder/internal/api/users"
+	"github.com/edulinq/autograder/internal/util"
+)
+
+func ListServerUsersTable(response core.APIResponse) string {
```

Comment:
Does anyone else us this function?
If not, then it should probably just be non-exported in users-list.